### DISCUSSION TITLE
Unable to select fine tuned model

### DISCUSSION THREAD

**Discussion by borishar** [erew123/alltalk_tts#158] [Q&A] [answered]
![download (2)](https://github.com/erew123/alltalk_tts/assets/6139836/1b7efcdb-d90b-4e1b-8c0c-cd6e73aa58bf)

guys, i finetuned the model, moved it with the provided option, the all talk standalone says detected during start, but I can't see the radio button to select it.. what am I doing wrong?

**Comment by erew123** [comment, marked as answer]
Hi @borishar 

Currently, there is no extra loader for the finetuned models outside of Text-gen-webui or SillyTavern. I have not had time to build additional loaders in other places within the interface, though it is on my [to-do-list](https://github.com/erew123/alltalk_tts/discussions/74) (please see the General section). However I am thinking through a more dynamic model loading system that not only allows additional models, but also other TTS engines. 

As for your finetuned model, unless you are using it within Text-gen-webui or SillyTavern, in which case you will see a additional load box for a finetuned model in the respective interfaces, you can:

1) Manually load it with the API from a command line https://github.com/erew123/alltalk_tts?tab=readme-ov-file#-switching-model-endpoint as long as its in the correct folder (As detailed in the API).

2) Copy your trained model over the base model and load it as your base model.

--- 

![image](https://github.com/erew123/alltalk_tts/assets/35898566/e759161b-595d-47c4-89f4-c0d6de8203cd)

---

Thanks
